packages/web-api/src/WebClient.spec.js

```diff
@@ -278,7 +278,7 @@ describe('WebClient', function () {
           .map((v) => ({ method, args: Object.assign({}, v, args) }))
         return acc.concat(textPatterns)
       }, []).forEach(({ method, args }) => {
-        it(`should send both text and fallback-argument-specific warning to logs when client executes ${method} without text argument(${args.text === "" ? "empty" : args.text}) nor without attachment-level fallback argument (${args.attachments[0].fallback === "  " ? "empty" : args.attachments[0].fallback})`, function () {
```

Comment:
All the existing tests should behave the same. My intention here is that we should change the behavior only for the pattern with **no text + valid fallbacks for all attachments**, which is your situation. Can you add a new test pattern to cover your situation?